server/sonar-web/src/main/js/apps/projectActivity/components/ProjectActivityPageHeader.js

```diff
@@ -28,19 +29,29 @@ type Props = {
   category?: string
 };
 
+type State = {
+  options: Array<{ label: string, value: string }>
+};
+
 export default class ProjectActivityPageHeader extends React.PureComponent {
   props: Props;
+  state: State;
+
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      options: EVENT_TYPES.map(category => ({
```

Comment:
To not have to compute it every time in the render method, since it should never change.
But it would probably be better to put it as a constante or a field of the component.